sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/LikeSimplificationSuite.scala

```diff
@@ -61,6 +61,20 @@ class LikeSimplificationSuite extends PlanTest {
     comparePlans(optimized, correctAnswer)
   }
 
+  test("simplify Like into StartsAndEndsWith") {
```

Comment:
StartsAndEndsWith isn't an expression. just say simplify Like into startsWith and EndsWith